src/utils/signing.ts

```diff
@@ -29,6 +30,9 @@ const issueCredential = async (
         : ['https://www.w3.org/2018/credentials/v1'],
       type: type ? ['VerifiableCredential', type] : ['VerifiableCredential'],
       credentialSubject: { id: sub, ...claimToObject(claims) },
+      credentialSchema: credentialSchemaId
+        ? { id: credentialSchemaId, type: 'JsonSchemaValidator2018' }
```

Comment:
@mirceanis I put "JsonSchemaValidator2018" here because it's the only thing I found in an example, but I'm not sure it's right. Does it seem right to you?

Reply:
It seems right but I'm also new to the space of VC schemas :)